packages/wp-now/src/wp-now.ts

```diff
@@ -278,6 +293,22 @@ async function initWordPress(
 	php.setPhpIniEntry('auto_prepend_file', configFile);
 }
 
+async function activatePluginOrTheme(
+	php: NodePHP,
+	{ projectPath, mode }: WPNowOptions
+) {
+	if (mode === WPNowMode.PLUGIN) {
+		const pluginFile = getPluginFile(projectPath);
+		console.log('pluginFile', pluginFile);
+		if (pluginFile) {
+			await activatePlugin(php, { pluginPath: pluginFile });
```

Comment:
Should we always call `activatePlugin()`, so it's obvious to the user when this isn't working?

Reply:
Removed this condition.
https://github.com/WordPress/wordpress-playground/pull/351/commits/659d75ecda100eaa5732d9cdcb800d4bebff944e

Currently, there will always be a pluginFile detected for Plugin mode.